platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/CryptoStatic.java

```diff
@@ -247,17 +247,10 @@ public static KeyStore createEmptyTrustStore() throws KeyStoreException {
     }
 
     /**
-     * check whether the given signature is valid
-     *
-     * @param data
-     * 		the data that was signed
-     * @param signature
-     * 		the claimed signature of that data
-     * @param publicKey
-     * 		the claimed public key used to generate that signature
-     * @return true if the signature is valid
+     * See {@link SignatureVerifier#verifySignature(byte[], byte[], PublicKey)}
      */
-    public static boolean verifySignature(byte[] data, byte[] signature, PublicKey publicKey) {
+    public static boolean verifySignature(
+            @NonNull byte[] data, @NonNull byte[] signature, @NonNull PublicKey publicKey) {
```

Comment:
Objects.requireNonNull() checks before passing the arguments into other methods.

Reply:
done